### PR TITLE
{bio}[goolf/1.7.20] FLASH2 v20160507

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/FLASH2/FLASH2-20160507-goolf-1.7.2064826f4.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+]
+
+files_to_copy = [(['flash2'], 'bin'), 'README']
+                
+sanity_check_paths = {
+    'files': ['bin/flash2'],
+    'dirs': [],
+}
+
+modloadmsg = "This app uses all available cpus in the machine by default!! PLEASE BE SURE TO USE THE --threads FLAG!"
```
Comment (boegel) [inline easybuild/easyconfigs/f/FLASH2/FLASH2-20160507-goolf-1.7.2064826f4.eb]: what does `--threads` do? why not mention that (it's OK if not everyone 100% understands imho)

File: easybuild/easyconfigs/f/FLASH2/FLASH2-20160507-goolf-1.7.2064826f4.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'FLASH2'
+version = '20160507'
+versionsuffix = '64826f4' # git commit to download
```
Comment (boegel) [inline easybuild/easyconfigs/f/FLASH2/FLASH2-20160507-goolf-1.7.2064826f4.eb]: @pescobar what's the point in exposing this?

why not:

```python
commit = '64826f4'
...
sources = ['%s.tar.gz' % commit]
```

Of course, the real 'fix' would be to support checking out Git repositories as a download mechanism...

**Comment by fgeorgatos** [comment]
Exposing it is potentially permitting --try-versionsuffix=bla , no?!

In easybuild/easyconfigs/f/FLASH2/FLASH2-20160507-goolf-1.7.2064826f4.eb
<https://github.com/hpcugent/easybuild-easyconfigs/pull/4355#discussion_r106813269>
:

> > @@ -0,0 +1,38 @@
> +# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
> +# Author: Pablo Escobar Lopez
> +# sciCORE - University of Basel
> +# SIB Swiss Institute of Bioinformatics
> +
> +easyblock = 'MakeCp'
> +
> +name = 'FLASH2'
> +version = '20160507'
> +versionsuffix = '64826f4' # git commit to download
>
> @pescobar <https://github.com/pescobar> what's the point in exposing this?
>
-- 
echo "sysadmin know better bash than english"|sed s/min/mins/ \
  | sed 's/better bash/bash better/' # signal detected in a CERN forum

**Comment by pescobar** [comment]
besides allowing `--try-versionsuffix=bla` using the commit ID in `versionsuffix` allows that the module name contains the commit id (which I find convenient)

**Comment by boegel** [comment]
@pescobar I don't understand why having the commit included in the module name is convenient... Doesn't exposing the commit ID to users cause confusion?

W.r.t. being able to use `--try-versionsuffix`, maybe we should add `commit` as a known easyconfig parameter in the EasyBuild framework, along with an accompanying `--try-commit`, where the `version` would be auto-derived to be the datestamp of the commit?

**Comment by fgeorgatos** [comment]
I

On Mon, 20 Mar 2017 at 8:08 PM Kenneth Hoste <notifications@github.com>
wrote:

> @pescobar <https://github.com/pescobar> I don't understand why having the
> commit included in the module name is convenient... Doesn't exposing the
> commit ID to users cause confusion?
>
> W.r.t. being able to use --try-versionsuffix, maybe we should add commit
> as a known easyconfig parameter in the EasyBuild framework, along with an
> accompanying --try-commit, where the version would be auto-derived to be
> the datestamp of
>
-- 
echo "sysadmin know better bash than english"|sed s/min/mins/ \
  | sed 's/better bash/bash better/' # signal detected in a CERN forum

**Comment by fgeorgatos** [comment]
I've been there and have also appreciated the one liner elegant approach.
I'd rather keep things as they have.
-- 
echo "sysadmin know better bash than english"|sed s/min/mins/ \
  | sed 's/better bash/bash better/' # signal detected in a CERN forum

File: easybuild/easyconfigs/f/FLASH2/FLASH2-20160507-goolf-1.7.2064826f4.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/f/FLASH2/FLASH2-20160507-goolf-1.7.2064826f4.eb]: We have an alternate approach for this now, i.e. to rename the source tarball on download, see http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): uses a deprecated toolchain.
For more information on deprecated toolchains, see https://easybuild.readthedocs.io/en/latest/Deprecated-easyconfigs.html#deprecated-toolchains.
Please consider reopening or submitting a new PR using a supported toolchain.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**